src/react-skycons.js

```diff
@@ -44,6 +44,9 @@ const ReactSkycons = React.createClass({
    this.state.skycons.set(React.findDOMNode(this), Skycons[nextProps.icon]);
   },
 
+  componentWillUnmount: function componentWillUnmount() {
+    this.state.skycons.remove(React.findDOMNode(this));
```

Comment:
I agree, `this.state.skycons.pause()` will work, however I think the API should expose this since this feels like more of a hack than anything